backend/global_var.py

```diff
@@ -212,6 +215,7 @@ def __init__(
         country: str,
         utcoffset: Union[float, int],
         crawler_code: str,
+        additional_info: List[Tuple[str, str]],
```

Comment:
If we don't append additional info (like I propose on previous comment). We can write a `@property` and retrieve it by the key of region (or country code)